java/component/keyple-core/src/main/java/org/eclipse/keyple/core/plugin/reader/AbstractObservableLocalReader.java

```diff
@@ -113,6 +113,12 @@
   /* Service that handles Internal Events and their impact on the current state of the reader */
   protected final ObservableReaderStateService stateService;
 
+  /** Method called when the NFC detection is started by the Keyple Plugin */
```

Comment:
The term NFC is misleading here because it could refer to Android NFC when the concept may be common to other technologies.
I would prefer "card detection".